harness/determined/pytorch/_pytorch_context.py

```diff
@@ -800,6 +803,29 @@ def current_train_batch(self) -> int:
             raise det.errors.InternalException("Training hasn't started.")
         return self._current_batch_idx
 
+    def get_tensorboard_writer(self) -> Any:
+        """
+        Return the context's tensorboard writer to the user if the user needs it.
+
+        This method ensures we create a writer only when necessary
+        and imports SummaryWriter at runtime.
+
+        The return type is Any to avoid creating an environment dependency
+        on the tensorboard module, which is required to import SummaryWriter.
```

Comment:
This docstring reads like it's written for other developers, but this object is part of the user-facing API and so it should be targeted at end-users.